meta-ostro/conf/distro/include/stateless.inc

```diff
@@ -96,9 +97,12 @@ APPEND_remove = "ro"
 STATELESS_MV_ROOTFS += "fstab"
 STATELESS_ETC_WHITELIST += "fstab"
 
-# TODO: systemd must be patched to load Smack rules also from elsewhere
-# than /etc/smack.
-STATELESS_ETC_WHITELIST += "smack/*"
```

Comment:
Don't bother updating STATELESS_ETC_WHITELIST. It just causes unnecessary recompilations.